src/core/service.c

```diff
@@ -77,6 +78,7 @@ static const UnitActiveState state_translation_table[_SERVICE_STATE_MAX] = {
         [SERVICE_AUTO_RESTART]               = UNIT_ACTIVATING,
         [SERVICE_AUTO_RESTART_QUEUED]        = UNIT_ACTIVATING,
         [SERVICE_CLEANING]                   = UNIT_MAINTENANCE,
+        [SERVICE_MOUNTING]                   = UNIT_MAINTENANCE,
```

Comment:
No. Maintenance means the unit is inactive and cannot be activated just yet, due to finalizing work.

Reply:
This was suggested by @poettering

Reply:
Where? But anyways, until now we definitely don't treat UNIT_MAINTENANCE as an active state.

Reply:
i don't think i suggested exposing this as UNIT_MAINTENANCE, did i?

i think we should consider the low level stat as a high-level UNIT_RELOADING type. It is in a way the applicaiton of new configuration to the unit as it is live, and I think that fits well enough.

Reply:
or hmm, i take that back. we colaesce reload requests, it would be bad if we'd create a concept of two concept of reloading which cannot reasonably be coalesced.

hence, please add a new high-level state for this. UNIT_READJUSTING or so. Should be more generic than "MOUNTING" i think, maybe we want to stick other "live" reconfigurations under this too.

UNIT_ADAPTING?

UNIT_REALINING?

Reply:
And please add the new high-level state to `UNIT_IS_ACTIVE_OR_RELOADING`